sycl/source/detail/device_info.hpp

```diff
@@ -568,6 +568,47 @@ struct get_device_info_impl<range<Dimensions>,
   }
 };
 
+#define ARCHES(X)                                                              \
```

Comment:
Can you please add a comment indicating that this macro is for AMD and NVIDIA architectures only?

Reply:
Done (Intel included)